src/js/game/items/boolean_item.js

```diff
@@ -48,15 +50,19 @@ export class BooleanItem extends BaseItem {
      * @param {DrawParameters} parameters
      */
     drawItemCenteredImpl(x, y, parameters, diameter = globalConfig.defaultItemDiameter) {
-        let sprite;
-        if (this.value) {
-            sprite = Loader.getSprite("sprites/wires/boolean_true.png");
-        } else {
-            sprite = Loader.getSprite("sprites/wires/boolean_false.png");
-        }
+        const value = Boolean(this.value).toString();
```

Comment:
I like this PR, except this part: I prefer manually specifying `true` and `false` for the string, so it's more clear which sprite is used.